### PR TITLE
fix(translation): german language

### PR DESCRIPTION
**Pull request by YunaBraska**
Fix some obvious translation issues. :)

### CODE REVIEW

File: scripts/translate/de_DE.po [hunk 1]
```diff
@@ -1868,7 +1868,7 @@ msgstr "Registrieren"
 #: menus/network.lua:919
 #, fuzzy
 msgid "~!Join Local Game"
-msgstr "~! Treten Sie dem lokalen Spiel bei"
+msgstr "~! Lokalen Spiel beitreten"
```
Comment (Jarod42) [inline scripts/translate/de_DE.po:1871]: Should it be 
```suggestion
msgstr "Lokalen Spiel beitreten (~!J)"
```

File: scripts/translate/de_DE.po [hunk 2]
```diff
@@ -2536,7 +2536,7 @@ msgstr "~! EILE"
 #: orc/buttons.lua:230
 #, fuzzy
 msgid "~!RAISE DEAD"
-msgstr "~! TOTE AUFERWECKEN"
+msgstr "~! TOTE ERWECKEN"
```
Comment (Jarod42) [inline scripts/translate/de_DE.po:2539]:
```suggestion
msgstr "TOTE ERWECKEN (~!R)"
```

**Review by Jarod42 — COMMENTED** [review]
Note that I didn't understand German at all...

File: scripts/translate/de_DE.po (hunk 2 above)
Comment (Jarod42) [reply scripts/translate/de_DE.po:2539]: If I am correct, it is used to show shortcut/hotkey. And there are not adapted to translation...
So mismatch might be seen as "error".

Comment (YunaBraska) [reply scripts/translate/de_DE.po]: All right. Based on the logic, that `~!` is a prefix marker for hotkeys, I fixed some more hotkeys. :)

**Pull request merged**